cmd/terramate/cli/cloud.go

```diff
@@ -234,6 +237,63 @@ func (c *cli) cloudInfo() {
 	c.cloud.output.MsgStdOutV("next token refresh in: %s", time.Until(c.cred().ExpireAt()))
 }
 
+func (c *cli) cloudDriftShow() {
+	err := c.setupCloudConfig()
+	if err != nil {
```

Comment:
Not to be addressed in this PR, but a nitpick, because it's visible here: It would be nice to have a consistent scheme for user-facing error messages. Here we have 3-4 different styles right next to each other (just err, lowercase informal, uppercase proper with/without trailing colon).